### ISSUE TITLE
Close DHCP server correct after bound

### ISSUE THREAD

**Issue by wake-0** [infusion/node-dhcp#36]
Hello all!

How should the server correctly be closed after an IP address was bound?
I actually added it to the `on bound` but not sure if this is the correct place, because the following error appears "Error: Not running." Without the `server.close()` in the `on bound` no error appears but I am also not able to close the server :D

Example code:

```
server.on('bound', function(state) {
    console.log("bound ip address.");
    server.close();   
});
```
This shows the "Error: Not running".

```
process.on('uncaughtException', (err) => {
    console.log(err.stack);
});
```

**Comment by Blackjack92** [comment]
Yeah exactly. I created a test, where I assign a single ip address via dhcp and afterwards the server can be stopped. --> If the DHCP server is closed like described above I get a `"Error: not running"`. Can this behaviour be fixed?

**Comment by isarantis** [comment]
Any news about that? i would like to make something similar to start/stop the service when i want.

**Comment by infusion** [comment]
The start/stop mechanism is coming with the next release
